### DISCUSSION TITLE
Looking to sort swell product by recency

### DISCUSSION THREAD

**Discussion by Daniel-Alamezie** [swellstores/community#457] [Help] [answered]
Looking at the swell docs it seems like they have provided a way to sort product alphabetically asc/desc, but what if i wanted to sort it by date (so the most recently added products always show up at the top).
How would i go about achieving this or will i have to manually implement this sort?
![image](https://github.com/swellstores/community/assets/74880857/f5a0866c-08c6-462c-87fa-5276edb67adb)

**Comment by Daniel-Alamezie** [comment, marked as answer]
Nevermind it was as straightforward as  setting the sort type to  'date_created desc'

**Reply by ericingram** [reply]
Note that all collections are normally sorted by `id desc` by default, which is typically the same as `date_created desc` because the ID field is time-ordered (objectid).
